packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -422,11 +422,11 @@ class SnackBar extends StatefulWidget {
   /// (optional) The percentage threshold for action widget's width before it overflows
   /// to a new line.
   ///
-  /// Must be between 0 and 1. If the width of the snackbar's [content] is greater
-  /// than this percentage of the width of the snackbar less the width of its [action],
-  /// then the [action] will appear below the [content].
+  /// Must be between 0 and 1.
+  /// If the width of the snackbar's [action] divided by the width of the snackbar
+  /// is greater than this percentage then the [action] will appear below the [content].
   ///
-  /// At a value of 0, the action will not overflow to a new line.
+  /// At a value of 0, the action will always overflow to a new line.
```

Comment:
Just went to the [SnackBar example](https://main-api.flutter.dev/flutter/material/SnackBar-class.html#material.SnackBar.1) and added `actionOverflowThreshold: 0`

Sure enough, a value of zero always causes an overflow!